src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.ILLink.targets

```diff
@@ -79,13 +81,21 @@ Copyright (c) .NET Foundation. All rights reserved.
       <_DotNetHostFileName Condition=" '$(OS)' == 'Windows_NT' ">dotnet.exe</_DotNetHostFileName>
     </PropertyGroup>
 
-    <Delete Files="@(_LinkedResolvedFileToPublishCandidates)" />
-    <ILLink AssemblyPaths="@(_ManagedAssembliesToLink)"
+    <!-- Temporary workaround until the naming is finalized. Translate from TrimMode to Action metadata. -->
+    <ItemGroup>
+      <ManagedAssemblyToLink Condition=" '%(ManagedAssemblyToLink.TrimMode)' != '' ">
+        <Action>%(ManagedAssemblyToLink.TrimMode)</Action>
```

Comment:
The task still looks for `Action` metadata. Once the naming is finalized, I'll update the task and remove the workaround.

Reply:
I plan to merge this PR before https://github.com/mono/linker/pull/1292, then update the SDK in the linker repo so that integration tests pass there, and then remove this workaround when the linker change is in.